scripts/fix_rpm

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -exo pipefail
+
+# Remove outdated repo URLs
+if [[ "$DISTRO_VER" == "7" ]]; then
+  rm -rf /etc/yum.repos.d/CentOS-*;
+  if [[ "$(uname -m)" == "x86_64" ]]; then
+    cp /tmp/centos7-x86_64-vault.repo /etc/yum.repos.d/vault.repo
+  else
+    cp /tmp/centos7-altarch-vault.repo /etc/yum.repos.d/vault.repo
+  fi
+  # Need to remove again because of some cache. Not sure where
```

Comment:
No cache.
The `yum update` just restores the deleted files because it updates them to include 7.9.2009 vault is not in the base image.
Hence it's better to disable; see https://github.com/conda-forge/docker-images/pull/271/files#r1661459536